frontend/polyfills.js

```diff
@@ -10,3 +10,4 @@ import 'core-js/fn/string/includes';
 import 'core-js/fn/string/pad-start';
 import 'core-js/fn/string/starts-with';
 import 'url-polyfill';
+import './polyfill-mutation-observer';
```

Comment:
As far as I can tell, this isn't a proper polyfill, it's a mock needed only for making our tests pass.

Therefore, I wouldn't add it here.

Reply:
ok, agree